### DISCUSSION TITLE
Only count videos, dont count livestreams

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#600] [Ideas]
This youtuber is currently streaming, so when I scanned the last 2 videos from his channel I think it counted the live-stream as a video too, make it so that it only counts YT videos as a **VIDEO**

![image](https://user-images.githubusercontent.com/84712013/153009389-61712c05-7fbc-42fa-ab44-6c957301e802.png)

**Comment by ThioJoe** [comment]
Well the number entered determines how many to fetch in the first place. And there's no way to know if there are any streams before fetching. So the solution is just do another check for 3

**Reply by KendallDoesCoding** [reply]
Yeah, I was gonna do that, but instead of restarting the program, I went ahead and took the 1 video and reported the spam comments.

**Reply by ThioJoe** [reply]
Not sure if this is what you mean but remember you can enter 'X' at any prompt to go back to the main menu without restarting the whole program.

**Reply by KendallDoesCoding** [reply]
> Not sure if this is what you mean but remember you can enter 'X' at any prompt to go back to the main menu without restarting the whole program.

Oh, I didn't know that. Thanks

**Reply by ThioJoe** [reply]
It literally says it at the top of the main menu every time lol
